src/common/components/wallet-portfolio-chart/index.tsx

```diff
@@ -0,0 +1,260 @@
+import React, { useEffect, useState } from 'react'
+import ReactHighcharts from "react-highcharts";
+import { Theme } from "../../store/global/types";
+import { _t } from "../../i18n";
+import { marketChart } from '../../api/misc'
+import moment from "moment";
+
+export const HiveWalletPortfolioChart = (props: any) => {
+    const { theme } = props;
+
+    const [prices, setPrices] = useState([])
+    
+    useEffect(() => {
+        marketChartInfo();
+    }, []);
+
+    const marketChartInfo = async () => {
+        const data: any = await marketChart("hive")
+            console.log(data)
```

Comment:
Remove please

Reply:
Thanks @dkildar,  all console.log( ) would be removed, I'm still working on this issue, would clean all and refactor once I'm done